lib/streamlit/elements/markdown.py

```diff
@@ -36,7 +36,7 @@ def markdown(self, body: str, unsafe_allow_html: bool = False) -> "DeltaGenerato
 
             This also supports:
 
-            * Emoji shortcodes, such as `:+1:`  and `:sunglasses:`.
+            * Emoji shortcodes, such as ``:+1:``  and ``:sunglasses:``.
```

Comment:
Is a single quote not working for getting it in the code formatting style? If so, we might need to go through our docstrings to fix this everywhere since I see single quotes in quite a few more places.

Reply:
The content enclosed in single backticks in reStructuredText is rendered as italics: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#quick-syntax-overview

Inline code requires two backticks. 

> If so, we might need to go through our docstrings to fix this everywhere since I see single quotes in quite a few more places.

I can take that in a follow-up PR 👍

Reply:
sounds good 👍  no urgency from me on this, just something to eventually get fixed